### PR TITLE
Feat: tag value suggestion and switch clickhouse-go to native v2

### PR DESCRIPTION
**Pull request by makeavish**
**Description:** <Describe what has changed.>
Integrated tag value suggestion
Clickhouse-go is switched to native v2

**Link to tracking Issue:**  https://github.com/SigNoz/engineering-pod/issues/191

### CODE REVIEW

File: exporter/clickhouseexporter/writer.go [hunk 1]
```diff
@@ -161,13 +147,13 @@ func (w *SpanWriter) writeIndexBatch(batch []*Span) error {
 			span.TagsValues,
 			span.StatusCode,
 			span.GetReferences(),
-			NewNullString(span.ExternalHttpMethod),
```
Comment (ankitnayan) [inline exporter/clickhouseexporter/writer.go:164]: why is `NewNullString` removed @makeavish ?

Comment (makeavish) [reply exporter/clickhouseexporter/writer.go:164]: clickhouse-v2 handles that automatically, don't have to use it

Comment (makeavish) [reply exporter/clickhouseexporter/writer.go:164]: Let me also clean the code by removing nullstring func _(with Axolo)_

Comment (ankitnayan) [reply exporter/clickhouseexporter/writer.go:164]: ok _(with Axolo)_

**Comment by makeavish** [comment]
@ankitnayan Please merge this if you have reviewed it

**Comment by ankitnayan** [comment]
@makeavish is the `test.go` file relevant anymore? Rest everything is okay

**Comment by makeavish** [comment]
Nope, `test.go` is not required anymore. Removed it.

**Pull request merged**